### ISSUE TITLE
Build AWS-CLI container image for s390 arch under Bare Metal platform

### ISSUE THREAD

**Issue by venkat-pinisetti** [red-hat-storage/ocs-ci#2435]
### Summary:
Running AWS-CLI container is  pre-requisite(setup stage) for couple of tests and currently using `amazon/aws-cli:2.0.13` for all the platforms which supports OCS-CI. However this container image was not working for s390 platform and causes test failure.

As part of workaround I found one image `seemethere/awscli-s390x` which supports s390 arch and pod also running without any issues.  

### Issues

1. Need to build aws-cli container image to support multi-arch.
2. Investigation required to use aws-cli container image in non-aws platforms such as s390x.
 

### Logs:

```
06:47:37 - MainThread - tests.helpers - INFO - Creating new Pod awscli-relay-pod for test
06:47:37 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding Pod with name awscli-relay-pod
06:47:37 - MainThread - ocs_ci.utility.templating - INFO - apiVersion: v1
kind: Pod
metadata:
  name: awscli-relay-pod
  namespace: default
spec:
  containers:
  - command:
    - /bin/sh
    image: amazon/aws-cli:2.0.13
    name: awscli
    stdin: true
    tty: true
    volumeMounts:
    - mountPath: /cert/service-ca.crt
      name: service-ca
      subPath: service-ca.crt
  volumes:
  - configMap:
      name: awscli-service-ca
    name: service-ca

06:47:37 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig create -f /tmp/POD_sx_r3shf -o yaml
06:47:37 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:47:38 - MainThread - ocs_ci.ocs.ocp - INFO - Waiting for a resource(s) of kind ConfigMap identified by name 'awscli-service-ca' using selector None at column name DATA to reach desired condition 1
06:47:38 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get ConfigMap awscli-service-ca -n default -o yaml
06:47:38 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get ConfigMap awscli-service-ca -n default
06:47:38 - MainThread - ocs_ci.ocs.ocp - INFO - Waiting for a resource(s) of kind Pod identified by name 'awscli-relay-pod' using selector None at column name STATUS to reach desired condition Running
06:47:38 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:47:39 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:47:39 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was ContainerCreating, but we were waiting for Running
06:47:39 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:47:42 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:47:42 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:47:42 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was ContainerCreating, but we were waiting for Running
06:47:42 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:47:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:47:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:47:46 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was ContainerCreating, but we were waiting for Running
06:47:46 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:47:49 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:47:49 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:47:49 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was ContainerCreating, but we were waiting for Running
06:47:49 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:47:52 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:47:52 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:47:53 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was ContainerCreating, but we were waiting for Running
06:47:53 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:47:56 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:47:56 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:47:56 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was ContainerCreating, but we were waiting for Running
06:47:56 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:47:59 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:47:59 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:48:00 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was ContainerCreating, but we were waiting for Running
06:48:00 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:48:03 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:48:03 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:48:03 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was ContainerCreating, but we were waiting for Running
06:48:03 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:48:06 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:48:06 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:48:07 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was CrashLoopBackOff, but we were waiting for Running
06:48:07 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:48:10 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:48:10 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:48:10 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was CrashLoopBackOff, but we were waiting for Running
06:48:10 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:48:13 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:48:14 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:48:14 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was CrashLoopBackOff, but we were waiting for Running
06:48:14 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:48:17 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:48:17 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:48:17 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was CrashLoopBackOff, but we were waiting for Running
06:48:17 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:48:20 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:48:21 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:48:21 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was CrashLoopBackOff, but we were waiting for Running
06:48:21 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:48:24 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:48:24 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:48:24 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was Error, but we were waiting for Running
06:48:24 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:48:27 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:48:28 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:48:28 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was Error, but we were waiting for Running
06:48:28 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:48:31 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:48:31 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:48:31 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was Error, but we were waiting for Running
06:48:31 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:48:34 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:48:35 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:48:35 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was Error, but we were waiting for Running
06:48:35 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:48:38 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:48:38 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:48:38 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was CrashLoopBackOff, but we were waiting for Running
06:48:38 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:48:41 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:48:41 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:48:42 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was CrashLoopBackOff, but we were waiting for Running
06:48:42 - MainThread - ocs_ci.ocs.ocp - ERROR - timeout expired: Timed Out: (60,)
06:48:42 - MainThread - ocs_ci.ocs.ocp - ERROR - Wait for Pod resource awscli-relay-pod at column STATUS to reach desired condition Running failed, last actual status was CrashLoopBackOff
06:48:42 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig describe Pod awscli-relay-pod
06:48:42 - MainThread - ocs_ci.ocs.ocp - WARNING - Description of the resource(s) we were waiting for:
Name:         awscli-relay-pod
Namespace:    default
Priority:     0
Node:         worker-3.cluster6.ibm.com/9.20.220.90
Start Time:   Fri, 03 Jul 2020 06:47:37 +0100
Labels:       <none>
Annotations:  k8s.v1.cni.cncf.io/networks-status:
                [{
                    "name": "openshift-sdn",
                    "interface": "eth0",
                    "ips": [
                        "10.130.2.172"
                    ],
                    "dns": {},
                    "default-route": [
                        "10.130.2.1"
                    ]
                }]
Status:       Running
IP:           10.130.2.172
IPs:
  IP:  10.130.2.172
Containers:
  awscli:
    Container ID:  cri-o://f7b2321333cd9afec73d433c243950a13112f3062805761e474b9baaba0843bd
    Image:         amazon/aws-cli:2.0.13
    Image ID:      docker.io/amazon/aws-cli@sha256:7161e38c1ee9bf1706a0f2264157a2d0f090316022ea5b1c783c438572c1e698
    Port:          <none>
    Host Port:     <none>
    Command:
      /bin/sh
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Fri, 03 Jul 2020 06:48:20 +0100
      Finished:     Fri, 03 Jul 2020 06:48:20 +0100
    Ready:          False
    Restart Count:  2
    Environment:    <none>
    Mounts:
      /cert/service-ca.crt from service-ca (rw,path="service-ca.crt")
      /var/run/secrets/kubernetes.io/serviceaccount from default-token-9ddfq (ro)
Conditions:
  Type              Status
  Initialized       True 
  Ready             False 
  ContainersReady   False 
  PodScheduled      True 
Volumes:
  service-ca:
    Type:      ConfigMap (a volume populated by a ConfigMap)
    Name:      awscli-service-ca
    Optional:  false
  default-token-9ddfq:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  default-token-9ddfq
    Optional:    false
QoS Class:       BestEffort
Node-Selectors:  <none>
Tolerations:     node.kubernetes.io/not-ready:NoExecute for 300s
                 node.kubernetes.io/unreachable:NoExecute for 300s
Events:
  Type     Reason     Age                From                                Message
  ----     ------     ----               ----                                -------
  Normal   Scheduled  <unknown>          default-scheduler                   Successfully assigned default/awscli-relay-pod to worker-3.cluster6.ibm.com
  Normal   Pulling    62s                kubelet, worker-3.cluster6.ibm.com  Pulling image "amazon/aws-cli:2.0.13"
  Normal   Pulled     40s                kubelet, worker-3.cluster6.ibm.com  Successfully pulled image "amazon/aws-cli:2.0.13"
  Normal   Created    22s (x3 over 39s)  kubelet, worker-3.cluster6.ibm.com  Created container awscli
  Normal   Started    22s (x3 over 39s)  kubelet, worker-3.cluster6.ibm.com  Started container awscli
  Normal   Pulled     22s (x2 over 38s)  kubelet, worker-3.cluster6.ibm.com  Container image "amazon/aws-cli:2.0.13" already present on machine
  Warning  BackOff    6s (x4 over 37s)   kubelet, worker-3.cluster6.ibm.com  Back-off restarting failed container

06:48:42 - MainThread - tests.helpers - ERROR - Pod awscli-relay-pod failed to reach Running
06:48:42 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:48:42 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig describe Pod awscli-relay-pod
ERROR                                                                                                                                                                 [100%]
----------------------------------------------------------------------------- live log teardown -----------------------------------------------------------------------------
```

### Workaround:
```
(.venv) [root@ocplnx51 ocs-ci]# git diff ocs_ci/templates/app-pods/awscli.yaml
diff --git a/ocs_ci/templates/app-pods/awscli.yaml b/ocs_ci/templates/app-pods/awscli.yaml
index da61a3a8..62577649 100644
--- a/ocs_ci/templates/app-pods/awscli.yaml
+++ b/ocs_ci/templates/app-pods/awscli.yaml
@@ -6,7 +6,7 @@ metadata:
 spec:
   containers:
     - name: awscli
-      image: amazon/aws-cli:2.0.13
+      image: seemethere/awscli-s390x
       # Override the default `aws` entrypoint in order to 
       # allow the pod to run continuously and act as a relay
       command: ['/bin/sh']
(.venv) [root@ocplnx51 ocs-ci]# 
```
```
06:40:49 - MainThread - tests.helpers - INFO - Creating new Pod awscli-relay-pod for test
06:40:49 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding Pod with name awscli-relay-pod
06:40:49 - MainThread - ocs_ci.utility.templating - INFO - apiVersion: v1
kind: Pod
metadata:
  name: awscli-relay-pod
  namespace: default
spec:
  containers:
  - command:
    - /bin/sh
    image: seemethere/awscli-s390x
    name: awscli
    stdin: true
    tty: true
    volumeMounts:
    - mountPath: /cert/service-ca.crt
      name: service-ca
      subPath: service-ca.crt
  volumes:
  - configMap:
      name: awscli-service-ca
    name: service-ca

06:40:49 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig create -f /tmp/POD_73hp9tyh -o yaml
06:40:50 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:40:50 - MainThread - ocs_ci.ocs.ocp - INFO - Waiting for a resource(s) of kind ConfigMap identified by name 'awscli-service-ca' using selector None at column name DATA to reach desired condition 1
06:40:50 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get ConfigMap awscli-service-ca -n default -o yaml
06:40:50 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get ConfigMap awscli-service-ca -n default
06:40:50 - MainThread - ocs_ci.ocs.ocp - INFO - Waiting for a resource(s) of kind Pod identified by name 'awscli-relay-pod' using selector None at column name STATUS to reach desired condition Running
06:40:50 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:40:51 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:40:51 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was ContainerCreating, but we were waiting for Running
06:40:51 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:40:54 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:40:54 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:40:54 - MainThread - ocs_ci.ocs.ocp - INFO - status of awscli-relay-pod at column STATUS was ContainerCreating, but we were waiting for Running
06:40:54 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
06:40:57 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default -o yaml
06:40:58 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /root/ocp4-workdir-ocs/auth/kubeconfig get Pod awscli-relay-pod -n default
06:40:58 - MainThread - tests.helpers - INFO - Pod awscli-relay-pod reached state Running
------------------------------------------------------------------------------- live log call -------------------------------------------------------------------------------
```

**Comment by SaravanaStorageNetwork** [comment]
I am working on providing a s390x based container image, will update once builds done.

**Comment by SaravanaStorageNetwork** [comment]
@venkat-pinisetti

Builds done.
Please use the image:  **quay.io/multiarch-origin-e2e/aws-cli:latest**

**Comment by MeridianExplorer** [comment]
Closed by #2582
